tendrl/commons/jobs/__init__.py

```diff
@@ -217,9 +209,8 @@ def process_job(job):
             )
             the_flow.run()
             try:
-                etcd_utils.write(job_status_key,
-                                 "finished",
-                                 prevValue="processing")
```

Comment:
Please use these etcd_util writes here, we need that `prevValue="new"` check ,

make the `status` and `locked_by` watchable

Reply:
I can make status and locked_by watchable attributes I feel. Will try that.